setup.py

```diff
@@ -64,7 +64,7 @@
         'test': [
             'codecov',
             'flake8',
-            'pytest>=3.0',
+            'pytest>=3.0,<4.0',
```

Comment:
related to this issue from `pytest-splinter` https://github.com/pytest-dev/pytest-splinter/issues/112